Silk.NET.sln

```diff
@@ -227,104 +227,6 @@ ProjectSection(SolutionItems) = preProject
 	examples\F#\OpenGL\Directory.Build.props = examples\F#\OpenGL\Directory.Build.props
 EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tutorial 1.1 - Hello Window", "examples\C#\OpenGL\Tutorial 1.1 - Hello Window\Tutorial 1.1 - Hello Window.csproj", "{13056B96-6B67-47E8-AFA5-3940C2D7F341}"
```

Comment:
Ensure you add the projects back

Reply:
I will add them back once they have been made, but they werent even showing up before, something was corrupted or something... idk.